swagger.yaml

```diff
@@ -4137,6 +4138,48 @@ definitions:
         type: string
         description: The device path.
 
+  domains_resolve_request:
+    type: object
+    required:
+      - domains
+    properties:
+      domains:
+        type: array
+        items:
+          $ref: '#/definitions/domain-resolve-request'
```

Comment:
Why do we need an object, why not just an array of strings?

Reply:
Because we might want to add more information to each domain (some metadata) in the future without breaking backwards compat